### DISCUSSION TITLE
EDC Connector - endpoint "api/v1/ids/data" throws 403 error

### DISCUSSION THREAD

**Discussion by tsatardekar-deloitte** [eclipse-edc/Connector#3384] [Q&A]
Hello Team,

I'm using connector as a docker image for setting up EDC Connector in the Mobility Data Space. I have setup the connector & also the EDC UI. In the EDC UI, in catalogue browser, when I enter the connector endpoint (https://XXX/api/v1/ids/data) in the search bar, I get the below error

![Screenshot 2023-08-23 at 14 45 12](https://github.com/eclipse-edc/Connector/assets/110841776/610ed59f-733d-4964-86b8-877f21edf587)

As I'm just using the connector as a docker image it is hard for me to understand what is happening inside. I also enabled debug logs on the connector and I see the below in the k8s pod logs

![Screenshot 2023-08-23 at 14 41 34](https://github.com/eclipse-edc/Connector/assets/110841776/4b409938-eced-4402-be4a-3795277ff4f5)

Can you please guide me here?

Thanks,
Trupti

**Comment by ndr-brt** [comment]
you are not supposed to call the protocol endpoints, these are meant for connector-to-connector communication.
to interact with the connector please use the [management api](https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.2.1).

**Comment by tsatardekar-deloitte** [comment]
Hello,

I'm not calling the protocol endpoint. It is used as an input on the catalogue browser as shown in the screenshot below
![Screenshot 2023-08-23 at 15 24 51](https://github.com/eclipse-edc/Connector/assets/110841776/485f6a12-abf8-402b-b905-6077e0ac68b5)

In the network tab of the browser, this endpoint being called 
https://XX/api/v1/management/catalog?providerUrl=https://XX/api/v1/ids/data&limit=10000000
and it gets the below response
[
    {
        "message": "org.eclipse.edc.spi.EdcException: Received an error from connector (https://XXX/api/v1/ids/data): 403 Forbidden",
        "type": "BadGateway",
        "path": null,
        "invalidValue": null
    }
]

**Reply by SebastianOpriel** [reply]
You address the wrong repository with your question. The current repository is about the Core EDC Connector framework. You use the specific Mobility Dataspace (MDS) version of EDC Connector + frontend. Please place your question here: https://github.com/sovity/edc-ui/discussions

**Reply by tsatardekar-deloitte** [reply]
Thank you. I will place my question there :)

**Comment by juliapampus** [comment]
This question does not address the EDC project but sovity's implementations for the MDS. Looks like something for you guys? @tmberthold @SebastianOpriel
